install_config/cluster_metrics.adoc

```diff
@@ -1018,7 +1019,8 @@ Prometheus].
 
 [NOTE]
 ====
-Metrics can be gathered from the `http://${POD_IP}:7575/metrics` endpoint.
+Once `openshift_metrics_project: openshift-infra` is installed, metrics can be
```

Comment:
@juzhao I'm not quite sure if this covers your request in https://github.com/openshift/openshift-docs/pull/5591. Is there anything else I can add here?

Reply:
@bmcelvee Yes, it covered my request in #5591. Thanks